packages/server-wallet/src/protocols/open-channel.ts

```diff
@@ -33,16 +33,13 @@ export type LedgerFundedAppState = AppState & {
 
 export type ProtocolState = DirectlyFundedAppState | LedgerFundedAppState;
 
-const stageGuard = (guardStage: Stage) => (s: State | undefined): s is State =>
-  !!s && stage(s) === guardStage;
+const isPreFundSetupSupported = (channelState: ChannelState): boolean =>
+  !!channelState.supported && channelState.supported.turnNum < channelState.participants.length;
 
-const isPrefundSetup = stageGuard('PrefundSetup');
-
-// These are currently unused, but will be used
-// const isRunning = stageGuard('Running');
-// const isPostfundSetup = stageGuard('PostfundSetup');
-const isRunning = stageGuard('Running');
-// const isMissing = (s: State | undefined): s is undefined => stage(s) === 'Missing';
+const isPostFundSetupSupported = (channelState: ChannelState): boolean =>
```

Comment:
This name is confusing. In a 2-party channel, state `2` can be supported, and state `2` is a post-fund setup. But if state `3` is not supported, then `isPostFundSetupSupported` would return `false`.

Reply:
How about `isEveryPostFundSetupStateSigned` or `isEverySupportPostFund` or `isSupportChainPostFunded`

Reply:
`isRunning` is exactly what the implementation is, which is what I would suggest.